packages/itwin/grouping-mapping-widget/src/widget/components/Grouping.tsx

```diff
@@ -314,6 +314,8 @@ export const Groupings = ({ mapping, goBack }: GroupsTreeProps) => {
     instance?: TableInstance,
   ): TableState => {
     switch (action.type) {
+      case "singleRowSelected":
```

Comment:
I am controlling the selection state here.
This feels kind of hacky though. Is row selection not suppose to return an action value? @mayank99